File: src/main/java/kickstart/UserAuthentication/LoginController.java
package kickstart.UserAuthentication;

import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;


@Controller
@RequestMapping(path = "/lunar_space_port")
public class LoginController {
	private UserRepository userRepository;



	public LoginController(UserRepository userRepository) {
		this.userRepository = userRepository;
	}



	@GetMapping(path = "/register")
	String register() {
		return "welcome";
		//userRepository.findByEmail()
	}


	@GetMapping(path = "/login")
	String login() {
		return "welcome";
		//userRepository.findByEmail()
	}


	@GetMapping(path = "/isLoggedIn")
	String isLoggedIn() {
		return "welcome";
	}


	@GetMapping(path = "/logOut")
	String logout() {
		return "welcome";
	}
}

File: src/main/java/kickstart/UserAuthentication/LoginController.kt
package kickstart.UserAuthentication




import org.springframework.http.MediaType
import org.springframework.stereotype.Controller
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping


@Controller
@RequestMapping("/lunar_space_port")
class LoginController () //private val userRepository: UserRepository)
{
    @GetMapping(path = ["/register"], produces = [MediaType.APPLICATION_JSON_VALUE])
    fun register(): String {
        return "welcome"
        //userRepository.findByEmail()
    }


    @GetMapping(value = ["/login"], produces = [MediaType.APPLICATION_JSON_VALUE])
    fun login() {
        //userRepository.findByEmail()
    }


    @GetMapping(path = ["/isLoggedIn"], produces = [MediaType.APPLICATION_JSON_VALUE])
    fun isLoggedIn(): String {
        return "welcome"
    }


    @GetMapping(path = ["/logOut"])
    fun logout(): String = "welcome"
}